docs/content/NewButton.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Buttons
+status: Alpha
+source: https://github.com/primer/react/tree/main/src/NewButton
+storybook: '/react/storybook?path=/story/composite-components-new-button--default-button'
+description: Button is core design component used for actions.
```

Comment:
Can we active voice here? [Related comment](https://github.com/primer/component-metadata/pull/3)

```
// Passive voice
Button is used for ...

// Active voice
Use button for ...
```